tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Kafka/IConsumerBuilder.cs

```diff
@@ -16,4 +17,6 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Kafka;
 internal interface IConsumerBuilder
 {
     IEnumerable<KeyValuePair<string, string>> Config { get; }
+
+    Action<object, object> OffsetsCommittedHandler { get; set; }
```

Comment:
It's interesting. This code does work, I verified with the test application

Reply:
> It's interesting. This code does work, I verified with the test application

I have to assume that you're not actually calling this code path in the application, because you can't cast `Action<IConsumer<TKey, TValue>, CommittedOffsets>` to `Action<object, object>` - it's not valid C# 🤷‍♂️ 😄 So your test application is working _despite_ this, not because of it 😉